chia/_tests/blockchain/test_blockchain.py

```diff
@@ -1810,46 +1815,39 @@ async def test_pre_validation(
     ) -> None:
         blocks = default_1000_blocks[:100]
         start = time.time()
```

Comment:
this test was changed. It's testing the pre-validation function itself. Previously it ensured that the list of blocks to be validated could be any length, without affecting the results. Since you can only pass in a single block at a time now, I changed the test to just validate all blocks once.